lib/chef/knife/bootstrap.rb

```diff
@@ -1094,6 +1112,18 @@ def session_timeout
 
         timeout.to_i
       end
+
+      def bootstrap_version_gt_15?
+        !!config[:bootstrap_version] &&
+          (config[:bootstrap_version] == "latest" ||
+           config[:bootstrap_version].split(".").first.to_i >= CHEF_15)
```

Comment:
`Chef::VersionString.new(config[:bootstrap_version]) >= 15.0`

probably need to `require_relative '../version_string' at the top as well